docs/HOW_TO_BUILD_BZZZ.md

```diff
@@ -0,0 +1,391 @@
+# How to Build bzzz
+This documentation includes the relevant information on how to build our quadcopter bzzz from scratch.
+
+## Tools
+You will need the following tolls to build bzzz.
+* Soldering iron with solder
+* Crimp tool
+* 1.5, 2, 2.5 Allen/Hex keys
+* 4mm flat and Phillips 2 screwdrivers
+* side cutting pliers
+* Pliers
+* X, Y, Z Spanners
```

Comment:
X, Y, Z?

Reply:
Need to check size of spanners we use/need